cedar-policy/src/api/id.rs

```diff
@@ -17,7 +17,7 @@
 //! This module defines the publicly exported identifier types including
 //! `EntityUid` and `PolicyId`.
 
-use crate::entities_json_errors::JsonDeserializationError;
+pub use crate::entities_json_errors::JsonDeserializationError;
```

Comment:
Can we avoid this `pub`? I think that will result in this type being exposed in two different places in `cedar-policy` (it's already public in the `entities_json_errors` module)